libqtile/notify.py

```diff
@@ -69,12 +76,16 @@ def Notify(self,  # noqa: N802, F722
             return self.manager.add(notif)
 
         @method()
-        def CloseNotification(self, _id: 'u'):  # type:ignore  # noqa: N802, F821
-            pass
+        def CloseNotification(self, nid: 'u'):  # type:ignore  # noqa: N802, F821
+            self.manager.close(nid)
 
         @signal()
-        def NotificationClosed(self, _id_in, _reason_in):  # noqa: N802
-            pass
+        def NotificationClosed(self, nid: 'u', reason: 'u') -> 'uu':  # noqa: N802
```

Comment:
Is this `noqa`s sufficient? I think you need `F821` too (for `undefined name`) and you may even need `type: ignore` for mypy.